web/controllers/role_controller.ex

```diff
@@ -7,7 +7,10 @@ defmodule CodeCorps.RoleController do
   plug :scrub_params, "data" when action in [:create]
 
   def index(conn, _params) do
-    roles = Repo.all(Role)
+    roles =
+      Role
+      |> Repo.all
+      |> Repo.preload([:skills])
```

Comment:
Does the `preload` need to happen before the `Repo.all` call? Or should we at least put it in front since thats how we do it for all other models?